### PR TITLE
Feature/2 project structure

### LINKED ISSUE #1: Add Builder for error codes with details in code
Add a builder like:

```
throw new IllegalArgumentException(ExceptionMessage.builder("E-STOLEN-BUCKIT-1")   
.message("Oh noes! They stole my buckit at {{time}} with {{fish}}.")
.description("...")
.parameter("time", new Date(), "at which time was is stolen")
.parameter("fish", 5, "how often was it stolen already")
.unquotedParameter("")
.mitigation("...")
.mitigation("...")
.mitigation("...")
.build())
```

### LINKED ISSUE #2: Create project structure

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #2

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>error-reporting-java</artifactId>
+    <version>0.1.0</version>
+    <name>error-reporting-java</name>
+    <description>Builder for exasol error codes..</description>
```
Comment (chiaradiamarcelo) [inline pom.xml]:
```suggestion
    <description>Builder for exasol error codes.</description>
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Java Error Reporting
+
+This package contains a Java-Builder for Exasol error messages.
```
Comment (chiaradiamarcelo) [inline README.md]:
```suggestion
This project contains a Java-Builder for Exasol error messages.
```

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+    </repositories>
+    <dependencies>
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest</artifactId>
+            <version>2.2</version>
```
Comment (chiaradiamarcelo) [inline pom.xml:83]: maybe we should add this to the properties section

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>-->
```
Comment (chiaradiamarcelo) [inline pom.xml:282]: I usually don't like commented out code. I understand that it is right now like this cause there's no release available of this plugin yet.

Consider removing this or leaving it, as you wish.

**Review by chiaradiamarcelo — APPROVED** [review]
Looks good to me, I just left some small comments

File: pom.xml (hunk 3 above)
Comment (jakobbraun) [reply pom.xml:83]: Since it is only used once, I see no advantage

**Pull request merged**